root/static/styles/layout.less

```diff
@@ -147,6 +147,8 @@ pre code {
 .align-left { text-align: left !important; }
 .align-top { vertical-align: top; }
 
+.clear-both { clear: both !important; }
```

Comment:
I added `!important` here to override `float: left` from the more-specific `form div.row` selector in root/static/styles/forms.less. I assume that anyone using this class actually wants it to take precedence, and there's precedent in the `align-left` class a few lines above this one. :-/